features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Applications/Applications.jsx

```diff
@@ -27,16 +27,16 @@ import { PageNotFound } from '../Base/Errors';
  * @class Applications
  * @extends {Component}
  */
-class Applications extends Component {
-    render() {
-        return (
-            <Switch>
-                <Route exact path='/applications' component={Listing} />
-                <Route path='/applications/:application_uuid/' render={props => <Details {...props} />} />
-                <Route component={PageNotFound} />
-            </Switch>
-        );
-    }
-}
 
-export default Applications;
+const applications = () => {
+    return (
+        <Switch>
+            <Route exact path='/applications' component={Listing} />
+            <Route path='/applications/:application_uuid/' render={props => <Details {...props} />} />
```

Comment:
This render method is redundant right ?
This is same as
```suggestion
            <Route path='/applications/:application_uuid/' component={Details} />
```
because when a component get rendered through `Route` it will get all the route related props passed by default

Reply:
fixed  https://github.com/wso2/carbon-apimgt/pull/6787/commits/b09339ab96cfdcc35f13e351833f406efad26ead